Examples/Sudoku-examples/CLIPS-puzzles/grid4x4-p13.clp

```diff
@@ -245,27 +240,28 @@ naked-single ==> r10c6 = 1
 naked-single ==> r12c11 = 2
 GRID 0 SOLVED. rating-type = W+SFin, MOST COMPLEX RULE TRIED = FSHP
 
-07 16 08 03 12 13 11 15 02 05 10 01 06 09 14 04
-05 01 06 13 10 14 07 04 08 03 15 09 16 12 02 11
-12 14 15 10 09 05 06 02 13 04 11 16 01 07 08 03
-04 09 11 02 01 08 16 03 14 12 06 07 10 05 15 13
-14 13 05 12 08 04 09 11 10 07 03 02 15 16 06 01
-16 04 03 01 02 07 10 12 06 15 14 05 13 08 11 09
-09 07 02 06 16 15 14 13 01 08 12 11 05 03 04 10
-15 08 10 11 06 03 01 05 04 16 09 13 12 14 07 02
-06 12 04 15 14 16 03 09 07 01 13 10 11 02 05 08
-02 03 16 14 07 01 15 10 12 11 05 08 04 13 09 06
-10 05 09 08 13 11 02 06 16 14 04 03 07 01 12 15
-01 11 13 07 05 12 04 08 09 06 02 15 03 10 16 14
-11 06 14 16 04 02 08 01 03 13 07 12 09 15 10 05
-13 02 01 09 15 06 12 14 05 10 16 04 08 11 03 07
-08 15 12 04 03 10 05 07 11 09 01 14 02 06 13 16
-03 10 07 05 11 09 13 16 15 02 08 06 14 04 01 12
+07 16 08 03 12 13 11 15 02 05 10 01 06 09 14 04 
+05 01 06 13 10 14 07 04 08 03 15 09 16 12 02 11 
+12 14 15 10 09 05 06 02 13 04 11 16 01 07 08 03 
+04 09 11 02 01 08 16 03 14 12 06 07 10 05 15 13 
+14 13 05 12 08 04 09 11 10 07 03 02 15 16 06 01 
+16 04 03 01 02 07 10 12 06 15 14 05 13 08 11 09 
+09 07 02 06 16 15 14 13 01 08 12 11 05 03 04 10 
+15 08 10 11 06 03 01 05 04 16 09 13 12 14 07 02 
+06 12 04 15 14 16 03 09 07 01 13 10 11 02 05 08 
+02 03 16 14 07 01 15 10 12 11 05 08 04 13 09 06 
+10 05 09 08 13 11 02 06 16 14 04 03 07 01 12 15 
+01 11 13 07 05 12 04 08 09 06 02 15 03 10 16 14 
+11 06 14 16 04 02 08 01 03 13 07 12 09 15 10 05 
+13 02 01 09 15 06 12 14 05 10 16 04 08 11 03 07 
+08 15 12 04 03 10 05 07 11 09 01 14 02 06 13 16 
+03 10 07 05 11 09 13 16 15 02 08 06 14 04 01 12 
```

Comment:
This is what I mean when I say the test validates down to the whitespace: CLIPS prints a trailing space on these lines, so the example must contain them too.